dhall-json/tasty/data/mismatchMessage0.json

```diff
@@ -0,0 +1,14 @@
+{
+    "a": {
+        "b": [
+            10,
+            0.5
+        ]
+    },
+    "c": {
+        "d": [
+            10,
+            0.5
+        ]
+    }
+}
```

Comment:
Is there a reason for the missing newlines in the .json files?

Reply:
Because I forgot it :smile: hehe